pydeps/mf27.py

```diff
@@ -38,18 +39,18 @@ def load_module(self, fqname, fp, pathname, file_info):
         # fqname = dotted module name we're loading
         suffix, mode, kind = file_info
         kstr = {
-            imp.PKG_DIRECTORY: 'PKG_DIRECTORY',
-            imp.PY_SOURCE: 'PY_SOURCE',
-            imp.PY_COMPILED: 'PY_COMPILED',
+            modulefinder._PKG_DIRECTORY: 'PKG_DIRECTORY',
```

Comment:
Apparently modulefinder doesn't export these constants until py3.8.

Reply:
Would you prefer that we redefine them ourselves? Do they change across different Python versions? :thinking:

Reply:
Only the magic number "constant" changes between python versions so it should be fine to define them. If you want to be extra defensive you could do
```
try:
    _PKG_DIRECTORY = modulefinder._PKG_DIRECTORY
    ...
except AttributeError:
    _PKG_DIRECTORY = 5
    ...etc
```